backend/api/Areas/Leases/Models/Lease/LeaseModel.cs

```diff
@@ -139,6 +139,26 @@ public class LeaseModel
         /// get/set - A list of insurance objects associated with this lease
         /// </summary>
         public IEnumerable<InsuranceModel> Insurances { get; set; }
+
+        /// <summary>
+        /// get/set - A collection of Improvements associated to this Lease
+        /// </summary>
+        public IEnumerable<PropertyImprovementModel> Improvements { get; set; }
+
+        /// <summary>
+        /// get/set - Whether this improvement contains a building that is subject to RTA.
+        /// </summary>
+        public bool IsSubjectToRta { get; set; }
+
+        /// <summary>
+        /// get/set - Whether this improvement contains a commercial building.
+        /// </summary>
+        public bool IsCommBldg { get; set; }
```

Comment:
nit: variable is not that long when is not abbreviated and makes it more readable "IsCommBldg -> isCommercialBuilding" thoughts?

Reply:
I'm reusing the same name as the one coming from the DB/Entity. Happy to rename but just know we have tons of these